charts/aws-ebs-csi-driver/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 1.3.0
 name: aws-ebs-csi-driver
 description: A Helm chart for AWS EBS CSI Driver
-version: 2.2.1
+version: 2.2.2
```

With a new feature please bump the version a minor.
```suggestion
version: 2.3.0
```

---

Done